.github/workflows/applicationTesting.yml

```diff
@@ -12,31 +12,48 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
-    strategy:
-      max-parallel: 4
-      matrix:
-        python-version: [3.6]
+
+    # https://hacksoft.io/github-actions-in-action-setting-up-django-and-postgres/
+    services:
+      postgres:
+        image: postgres:10.8
+        env:
+          POSTGRES_USER: ${{ secrets.POSTGRES_DB_USER }}
+          POSTGRES_PASSWORD: ${{ secrets.POSTGRES_DB_PASS }}
+          POSTGRES_DB: ${{ secrets.POSTGRES_DB_NAME }}
+        ports:
+          - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python ${{ matrix.python-version }}
+    - name: Set up Python python-version
       uses: actions/setup-python@v1
       with:
-        python-version: ${{ matrix.python-version }}
+        python-version: 3.6
+    
     
     - name: Install Dependencies
       run: |
         python -m pip install --upgrade pip
         pip install -r requirements/local.txt
-
+    
+    
+    - name: Run migrations
+      run: python manage.py migrate
+    
```

Comment:
the migration is runned in 2 steps , I think here it could be `upgrade` not `migrate`...

Reply:
this migration is the postgres service set at the beginning of the workflow so that the test runs properly.

Reply:
what do you mean by two 2 steps??

Reply:
This migration is the the postgres db set as service at the top so that we can have some tables to run the migration against.